MapboxVision.xcodeproj/project.pbxproj

```diff
@@ -1391,6 +1391,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
```

Comment:
Previously in order for test target to see framework's dependencies we used `Copy Files` phase as it is recommended by Carthage [here](https://github.com/Carthage/Carthage/blob/0.11/README.md#adding-frameworks-to-unit-tests-or-a-framework).

This approach also works and may even be easier to support (unless you forget about that while setting up a new test target) so let's use it. 

Please delete `Copy Files` phase then as it's also covered by this solution.